image/templates/helm/stackrox-central/README.md.htpl

```diff
@@ -174,6 +180,42 @@ helm upgrade -n stackrox stackrox-central-services stackrox/stackrox-central-ser
 Under most circumstances, you will not need to supply the `values-private.yaml` file, unless
 you want changes to sensitive configuration options to be applied.
 
-Of course you can also specify configuration values via the `--set` or `--set-file` command-line
-flags. However, these options will be forgotten with the next `helm upgrade` invocation, unless
-you supply them again.
+Alternatively, you can also use the `--reuse-values` for the `helm upgrade` command. This flag
+causes values which were previously passed via `--values / -f`, `--set` or `--set-file` to be
+automatially passed again to the Helm chart rendering.
+
+### Passing Generated Values
+
+The Helm chart can conveniently automatically generate certain sensitive data objects. This includes
+passwords, keys and certificates. In case such data objects were generated during the installation
+of the Helm chart, a command is provided as part of the post-installation notes which can be used
+for retrieving them and storing them in a file `generated-values.yaml`. In particular, this file
+might contain a CA key.
+
+If, during an upgrade of the Helm release, the Helm chart needs to generate a new certificate for
+some component it is required to pass in the existing CA, for example by adding
+`-f generated-values.yaml` to the arguments for `helm upgrade`.
+
+### Enabling Scanner V4
+
+As of StackRox 4.4 a new scanner component based on ClairCore named Scanner V4 is integrated.
+In a future version of StackRox this will become the new default scanner and completely replace the existing
+scanner named StackRox Scanner. For the time being Scanner V4 is not installed by default, but it
+can be enabled alongside StackRox Scanner using the following Helm chart setting:
```

Comment:
Hmm, I think this has a different meaning, but I guess it wasn't very clear what I have tried to convey. Let me explain.

First point: I did want to go into the details of the topic of running central-services with or without the old scanner. It is possible to disable the scanner on central side, but it is not recommended at all. I don't know how exactly such an environment would behave. I also don't know if people do that.

The second point: enabling scanner v4 without having the old scanner also enabled is totally unsupported and untested.

So, I wanted to explain how to enable scanner V4 and also state that it is only supposed to be enabled if the old scanner is also enabled. People should not be doing:

```
scanner:
  disable: true
scannerV4:
  disable: false
```

at least not at this point in time, maybe later.

But I have kind of attempted to ignore this edge case of running central without scanner, I didn't want to put too much attention to it -- didn't consider it worth the extra words for something that maybe nobody does.

So that's why I snuggled in this "alongside StackRox Scanner.

I think your version would suggest that you can enable both scanners with just `scannerV4.disable: false`, wouldn't it?

Reply:
discussed offline.